### ISSUE TITLE
Subdirectory to point to a branch

### ISSUE THREAD

**Issue by Ehesp** [QingWei-Li/docsify#153]
Hey!

I'm working on a repo, and there's currently a `v2` branch which where we'd like to have separate docs to master. Locally I can get this working, but when pushing it to the repo it doesn't work.

1) First I made a v2 directory in master, with an `index.html`, where I point the bathPath to the branch ([here](https://github.com/wix/react-native-navigation/tree/master/docs/v2 )):

```
basePath: 'https://raw.githubusercontent.com/wix/react-native-navigation/v2/docs/',
```

2) I make some files (no `index.html`) in the v2 branch ([here](https://github.com/wix/react-native-navigation/tree/v2/docs)).

3) Booting this locally I can now switch between the master docs (http://localhost:3000/#/) and the v2 docs (http://localhost:3000/v2/#/) fine.

4) Pushing this to the server, I get null content (https://wix.github.io/react-native-navigation/v2/#/). When I look at the network requests, it's trying to load the raw github content from:
https://wix.github.io/raw.githubusercontent.com/wix/react-native-navigation/v2/docs/README.md

For some reason the wix.github.io is prepended to the URL, causing a 404. 

Is this expected, or do I need to do something else? Cheers!

**Comment by QingWei-Li** [comment]
Fixed in `3.6.4`

**Comment by Ehesp** [comment]
Awesome cheers :D
